react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -133,6 +191,32 @@ class ToolbarButton extends Component {
     }
 
     /**
+     * Parses the props and state to find any popup that should be displayed
+     * and returns an object desribing how the popup should display.
+     *
+     * @private
+     * @returns {Object|null}
+     */
+    _getPopupDisplayConfiguration() {
+        const { button, tooltipPosition } = this.props;
+        const { popups, popupDisplay } = button;
```

Comment:
I see that the code was presuming that already, but which of our buttons support multiple popups? Have you checked?